workload/terraform/greenfield/AADscenario/readme.md

```diff
@@ -1,4 +1,4 @@
-# Implement Azure Virtual Desktop with Azure Active Directory
```

Comment:
"Azure" needs to be removed, the new service name is "Microsoft Entra ID" not "Azure Microsoft Entra ID"

Reply:
Done! @danycontre